src/main.cpp

```diff
@@ -4,5 +4,27 @@ int main(int argc, char * argv[]) {
     Controller controller = Controller(WIDTH, HEIGHT, ENERGY);
     controller.init();
     controller.run();
+
+    // debug terminalview render function
+    TerminalView terminalView = TerminalView();
+    World world = World(1000, 1000, 1000);
+
+    Fred * fred1 = new Fred(123, 321, 100);
+    Fred * fred2 = new Fred(138, 153, 70);
+    Fred * fred3 = new Fred(420, 420, 5);
+
+    Food * food1 = new Food(500, 689, 10);
+    Food * food2 = new Food(1000, 1000, 100);
+    Food * food3 = new Food(0, 0, 1);
+
+    world.addIndividual(fred1);
+    world.addIndividual(fred2);
+    world.addIndividual(fred3);
+
+    world.addFood(food1);
+    world.addFood(food2);
+    world.addFood(food3);
+
+    terminalView.render(world);
```

Comment:
why is this in the main.cpp file??

Reply:
the controller initialises a world with individuals and food.
all you need to do is incorporate it in the controller.
otherwise, just put it in test files.

But, please, refrain from putting debug stuff like this in our main prod code

Reply:
Janek told me to. I just wanted to put it in there to test stuff but he said I should also commit it.
All I heard from him is that the controller doesn't do anything

Reply:
Janek's not responsible for the controller.
If you want to test it in the main file, please put all this in the controller.

The controller initialises the world; you can just change the implementation of `.init()` to however you want to initialise the world.
Also, in `.simulate()`, call the `.render()` function, because that's exactly the place where things will be rendered in the future anyways.

Everything which currently is in `.init()` and `.simulate()` is only there for testing, you can just replace it by how you need it

Reply:
yeah listen to Aaron, I am unaware of the current state here :)

Reply:
commit 6ff2b4e